.circleci/config.yml

```
@@ -154,6 +154,22 @@ commands:
       - run:
           name: Post comment on GitHub
           command: ./scripts/notify-github.sh "<< parameters.data >>"
+  pack_and_build_sdk:
+    parameters:
+      working_directory:
+        type: string
+        default: ~/project
+    steps:
+      - install_node_modules
+      - run:
+          command: yarn build
+          working_directory: ~/project
+      - run:
+          command: yarn pack --filename InstabugSDK.tgz
+          working_directory: ~/project
+      - run:
+          command: yarn add file:../../InstabugSDK.tgz
+          working_directory: ~/project/examples/hybrid
```

Do we need to lock this to the hybrid example app? We might want to integrate the same command in other example apps in the future, what do you think?
We can probably use the `working_directory` passed for this and default the `working_directory` parameter to the `hybrid` example app.

```suggestion
          working_directory: <<parameters.working_directory>>
```